x-pack/plugins/watcher/public/components/action_type_select/action_type_select.js

```diff
@@ -30,7 +30,9 @@ app.directive('actionTypeSelect', function ($injector) {
 
         this.loadActionTypes()
           .then(actionTypes => {
-            this.actionTypes = actionTypes;
+            this.actionTypes = actionTypes.filter((actionType) => {
+              return actionType.typeName !== 'Action';
```

Comment:
Maybe worth a comment explaining why we're doing this?

```js
// 'Action' is the default action type. If an action has the default then it's
// a no-op action and shouldn't be presented to the user.
```

Reply:
Maybe at a later time we can re-evaluate this default and see if it even makes sense to have one. Seems more intuitive for it to default to `undefined`, which intuitively seems like it shouldn't be available in the UI.